gulpfile.babel.js

```diff
@@ -61,9 +61,22 @@ gulp.task('dist:config', () => {
 
 // Build webpack for production
 gulp.task('dist:webpack', (done) => {
+  // process request for static assets
+  if (process.env.PUBLIC_PATH) {
+    if (process.env.PUBLIC_PATH[0] !== '/') {
+      throw Error('PUBLIC_PATH must begin with /');
```

Comment:
This is weird. Shouldn't public path be able to be a full domain rather than a path?